src/core/server/connector/connector_manager.cc

```diff
@@ -0,0 +1,55 @@
+#include "core/server/connector/connector_manager.h"
+
+#include <vector>
+
+#include <gflags/gflags.h>
+#include <glog/logging.h>
+
+#include "core/frame_response.h"
+#include "core/server/connector/connector.h"
+#include "core/server/connector/human_connector.h"
+#include "core/player.h"
+
+#if defined(_MSC_VER)
+#include "core/server/connector/pipe_connector_win.h"
+#else
+#include "core/server/connector/pipe_connector_posix.h"
+#endif
+
+using namespace std;
+
+DEFINE_bool(realtime, true, "use realtime");
+DEFINE_bool(no_timeout, false, "if true, wait ai's thought without timeout");
+
+ConnectorManager::ConnectorManager(unique_ptr<Connector> p1, unique_ptr<Connector> p2, bool timeout)
+    : connectors_{ move(p1), move(p2) }
```

Comment:
Basically I put ':' in the previous line, and won't put ',' in the first character of the line.